extensions/data-workspace/src/common/constants.ts

```diff
@@ -20,10 +20,12 @@ export const EnterWorkspaceConfirmation = localize('dataworkspace.enterWorkspace
 export const WorkspaceContainsNotAddedProjects = localize('dataworkspace.workspaceContainsNotAddedProjects', "The current workspace contains one or more projects that have not been added to the workspace. Use the 'Open existing' dialog to add projects to the projects pane.");
 export const LaunchOpenExisitingDialog = localize('dataworkspace.launchOpenExistingDialog', "Launch Open existing dialog");
 export const DoNotShowAgain = localize('dataworkspace.doNotShowAgain', "Do not show again");
-export const ProjectsFailedToLoad = localize('dataworkspace.projectsFailedToLoad', "Some projects failed to load. Please open console for more information");
+export const ProjectsFailedToLoad = localize('dataworkspace.projectsFailedToLoad', "Some projects failed to load. Please turn on developer tools and view console for more information.");
```

Comment:
"turn on developer tools" isn't really accurate (and most people aren't going to know how to do that). 

I'd suggest using the command syntax to provide a link they can click to do this - it should be something like this - `[open the developer console](command:workbench.action.toggleDevTools)`

Reply:
Also is it really best to tell users to open the developer console anyways? Why not tell them to open the git output - I would expect that'd be a better thing to use since it'll contain all the logging from the git extension (not just the errors)